index.html

```diff
@@ -41,6 +41,19 @@ <h1 class="header-content__title__text">プレスリリース・ニュースリ
         </ul>
       </nav>
     </header>
+    <ul class="category-list">
+      <li class="category focused"><a href="#">総合</a></li>
```

Comment:
各`li`は`element`ですね。`focused`はmodifierになります
```suggestion
      <li class="category__item category__item--focused"><a href="#">総合</a></li>
```

Reply:
BEMの原則のことを失念してました、、、修正します！

Reply:
[こちらのcommitで修正いたしました！](https://github.com/shunsock/prtimes_htmlcss/pull/16/commits/4578866a13a73a02238433969b148ecf446ef474)